### ISSUE TITLE
Message : importance

### ISSUE THREAD

**Issue by redman11x** [jstedfast/MailKit#207]
Hello,

Is there any possibility to get the importance of the message (or the priority) ? I can't find it in the properties of the MimeMessage.

I use the latest version of MailKit.

Thanks a lot.

**Comment by jstedfast** [comment]
Note: technically this issue belongs in the [MimeKit](https://github.com/jstedfast/MimeKit) issue tracker...

There aren't any convenience properties for this in the currently released version, but you can get these values like this:

```csharp
var importance = message.Headers[HeaderId.Importance];
var priority = message.Headers[HeaderId.Priority];
```

That will give you the raw string values of those headers.

I just [committed a patch](https://github.com/jstedfast/MimeKit/commit/1b631b0916a67fa2ba8f6c07aa05951469b984fe) that adds properties to MimeMessage to get these values as enums, but I probably won't make another release for a week or 2.

Hope that helps.

**Comment by redman11x** [comment]
Thanks a lot for your support, it works !

**Comment by jstedfast** [comment]
I've released MimeKit and MailKit 1.2.5 that contains this fix.
